client_test.go

```diff
@@ -129,10 +129,11 @@ func TestMain(m *testing.M) {
 	}).Info("running tests against containerd")
 
 	// pull a seed image
+	log.G(ctx).Info("start to pull seed image")
 	if _, err = client.Pull(ctx, testImage, WithPullUnpack); err != nil {
-		ctrd.Stop()
-		ctrd.Wait()
 		fmt.Fprintf(os.Stderr, "%s: %s\n", err, buf.String())
```

Comment:
show the error first and then kill it.

Reply:
I'm going to pull this fix into #3222 because if this ever happens again, it causes quite a backup as it requires the full 60 minutes to expire before AppVeyor kills the run.